scaffolding-node/plan.sh

```diff
@@ -1,11 +1,11 @@
 pkg_name=scaffolding-node
 pkg_origin=core
-pkg_version="0.6.2"
+pkg_version="0.6.3"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('Apache-2.0')
 pkg_description="Habitat Plan Scaffolding for Node.js Applications"
 pkg_upstream_url="https://github.com/habitat-sh/core-plans/tree/master/scaffolding-node"
-pkg_deps=(core/tar core/jq-static core/rq)
+pkg_deps=(core/node core/tar core/jq-static core/rq)
```

Comment:
What if you were to put `core/node` into `$pkg_build_deps` instead? The Scaffolding implementation doesn't need this to run (it installs whatever version applies to the Node codebase), and any `$pkg_dep` for the Scaffolding will become a *build* dependency for all Node apps. Putting it in as a *build* dependency will trigger the Builder dependent rebuilds but not add any extra build or run dependencies for the Node apps.

Reply:
Certainly!  Getting on that now.